stored_queries/GO/GO_get_associated_ws_objects.yaml

```diff
@@ -0,0 +1,47 @@
+# Get the associated ws objects of this term
+
+name: GO_get_associated_ws_objects
+params:
+  type: object
+  required: [id, ts]
+  properties:
+    id:
+      type: string
+      title: Document ID
+      description: GO id of the term you want to get the children of
+    limit:
+      type: integer
+      title: Maximum result limit
+      default: 20
+      maximum: 1000
+    offset:
+      type: integer
+      title: Result offset for pagination
+      default: 0
+      maximum: 100000
+    ts:
+      type: integer
+      title: Versioning timestamp
+query_prefix: ws_genome_features, ws_object_version
+query: |
+  FOR t in GO_terms
+    FILTER t.id == @id
+    FILTER t.created <= @ts AND t.expired >= @ts
+    limit 1
+    FOR v, e, p IN 2 INBOUND t ws_feature_has_GO_annotation, ws_genome_has_feature
+      FILTER p.edges[0].created <= @ts AND p.edges[0].expired >= @ts
+      FILTER v.is_public
```

Comment:
Change this to `v.is_public OR v.workspace_id IN ws_ids`. The variable `ws_ids` is automatically set to the api to a list of private workspaces that the user has access to